sway/input/input-manager.c

```diff
@@ -738,15 +772,25 @@ void input_manager_configure_xcursor(void) {
 
 struct input_config *input_device_get_config(struct sway_input_device *device) {
 	struct input_config *wildcard_config = NULL;
+	struct input_config *type_config = NULL;
```

Comment:
This is not sufficient. This will need a somewhat simpler version of `get_output_config`